<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="main_button_create">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>361</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать схему и таблицы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_button_insert">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>361</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставить данные</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_button_show">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>361</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_label_notification">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
